postgres-consistency/scripts/EpochSlotToBlkHash.js

```diff
@@ -0,0 +1,82 @@
+var fetch = require("node-fetch");
+
+const serverRoute = 'https://explorer.iohkdev.io/api';
+
+const slotsPerEpoch = 21600;
+
+const randomNumberWithMax = function (topNumber) {
+  return Math.floor((Math.random() * topNumber));
+}
+
+function removeNullFromArray(actual) {
```

Comment:
I think this could help in order to simplify the code https://lodash.com/docs/4.17.10#compact